column_transforms/rasgo_concat/rasgo_concat.yaml

```diff
@@ -0,0 +1,21 @@
+name: rasgo_concat
+transform_type: column
+source_code: rasgo_concat.sql
+description: |
+  This function creates a new column that concatenates fixed values and columns in your dataset.
+
+  Pass in a list named "concat_list", containing the names of the columns and the static string values to concatenate, in order.
+
+arguments:
+  concat_list:
+    type: mixed_list
+    description: A list representing each new column to be created.
+example_code: |
+    rasgo.read.source_data(w_source.id, limit=5)
+
+    t1 = w_source.transform(
+      transform_name='rasgo_concat',
+      concat_list=["DS_WEATHER_ICON", "'some_str'", "'_5'"]
```

Comment:
will the single quotes be required on non-columns? is there a way to add those in the template so that we don't put that burden on the user?

Reply:
could be a future enhancement if it's difficult to add.